### PR TITLE
Upgrade to Quasar 0.15 (without new CLI)

### LINKED ISSUE #913: Upgrade to quasar 0.15
Lots of changes: https://github.com/quasarframework/quasar/releases
Upgrade and check if the page still looks the same and things work. Maybe a good chance to add more end-to-end tests?

### PR DESCRIPTION
**Pull request by tiltec**
Closes #913 

I upgraded Quasar to the latest version. It needed many changes, but it works and looks almost the same as before :smile: 

I didn't upgrade to the new CLI and the new project structure yet, as this requires moving and rewriting lots of files. But we might do it in future.

I didn't set up Quasar i18n, because we don't use any component that needs it: http://quasar-framework.org/components/internationalization.html

### CODE REVIEW

**Comment by nicksellen** [comment]
If we still need an eventbus you can make one with:

```js
const Events = new Vue()
```

File: src/store/plugins/presenceReporter.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ export default store => {
     .forEach(event => document.addEventListener(event, resetTimer))
 
   // listen on tab & window visibility, mostly works on desktop browser
-  Events.$on('app:visibility', state => {
+  /* Events.$on('app:visibility', state => {
```
Comment (tiltec) [inline src/store/plugins/presenceReporter.js]: Could be replaced with http://quasar-framework.org/components/app-visibility.html

Comment (tiltec) [reply src/store/plugins/presenceReporter.js]: ... but need to check if it also works without the quasar plugin

**Comment by tiltec** [comment]
I worked a bit more on it, solved the most obvious warnings and switched to the new QLayout component.

A few selected issues that I noticed:

- Toasts need to be replaced with notify
- Dialogs don't open
- AvailablePickups on group wall don't fold out
- Pickups don't fold out on pickup manage page
- Lots of styling issues everywhere, especially where we have custom styling on top of quasar components

I'll have a look at the Dialog component next, that's one of those new plugins that are supposed to be activated in the new `quasar.conf.js` file - which we would only get with the new Quasar CLI and the new directory structure. Maybe there's a way around it though.

**Comment by tiltec** [comment]
Functionality should be mostly done, design needs fixing.

**Comment by tiltec** [comment]
Ready for review! Have a look at https://deploy-preview-974--karrot.netlify.com/ and find the remaining problems :)

**Comment by nicksellen** [comment]
Great work!

newquasarui1

Two things:
1. content of the search box (the magnifying glass and the text "search" is aligned a bit low)
2. the close icon has fallen down a bit

**Comment by tiltec** [comment]
If there's no resistance until then, I'll merge tomorrow afternoon.

**Pull request merged**